### ISSUE TITLE
Work Queue Latency increased 10 times

### ISSUE THREAD

**Issue by eshikhov** [LINBIT/linstor-server#428]
When applying Linstore CRDs (group internal.linstor.linbit.com) Work Queue Latency increased 10 times

I rolled these CRDs into different clusters with the same effect everywhere

Only after the CRDs are removed, the delay of the work queue returns to normal time. On the graph you can see the moment of installing and removing CRDs.

![Снимок экрана 2024-11-27 в 18 33 21](https://github.com/user-attachments/assets/4448de2c-03a9-41f4-8248-ee9b9472f2d0)

Environment
    OS: Talos v1.8.3
    Kubernetes version: v1.30.7

**Comment by eshikhov** [comment]
Yes, you are correct, installing LINSTOR's CRDs makes k8s itself slower (higher worker queue latency).
Installing other CRDs, even more than LINSTOR's, has no negative impact.

**Comment by vladimirfx** [comment]
> The pure existence of LINSTOR's CRDs make k8s itself slower (higher worker queue latency).

Yes, you understand correctly. Only Linstore CRDs slow down `openapi_v3_contoller` in order of magnitude. Other CRDs (a few hundred) do not affect that controller. Unfortunately, I can't find relevant K8s issues.
What is interesting is that when CRDs are modified by removing old versions - the response time for `openapi_v3_contoller` is normalized.

**Comment by eshikhov** [comment]
Tried it in two different clusters
One distribution on talos Kubernetes version v1.30.7
the other on k3s Kubernetes version v1.23.16

On different distributions and versions of Kubernetes Work Queue Latency increased

**Comment by ghernadi** [comment]
Sorry for the silence.. we are investigating and will try to come up with a solution, but since our todo-lists are quite full, it might take a while.

Thanks anyways for reporting!
